.github/workflows/binary.yml

```diff
@@ -7,17 +7,20 @@ on:
   schedule:
     # Run every morning to detect flakiness and broken dependencies
     - cron: '17 8 * * *'
+env:
+  ROS_DISTRO: galactic
+  BRANCH: master
 
 jobs:
   ci_binary:
     name: Galactic binary job
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        env:
-          - {ROS_DISTRO: galactic, ROS_REPO: main}
-          - {ROS_DISTRO: galactic, ROS_REPO: testing}
+          ROS_REPO: [main, testing]
     steps:
-      - uses: actions/checkout@v1
+      - uses: actions/checkout@v2
       - uses: ros-industrial/industrial_ci@master
-        env: ${{matrix.env}}
+        env:
+          ROS_REPO: ${{matrix.ROS_REPO}}
+          ABICHECK_URL: github:ros-controls/ros2_control#${{ env.BRANCH }}
```

Comment:
I propose use of `github.base_ref` (see [here](https://docs.github.com/en/actions/learn-github-actions/contexts)). 

Also we only need this to run on pull request.